src/CommonProjectProperties.xml

```diff
@@ -0,0 +1,31 @@
+<Project>
+  <!-- Properties common to all projects under this directory -->
+
+  <PropertyGroup Condition="'$(OS)' == 'Windows_NT'">
+    <TargetFrameworks>$(TargetCoreFramework);$(TargetDesktopFramework)</TargetFrameworks>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(OS)' != 'Windows_NT'">
+    <TargetFramework>$(TargetCoreFramework)</TargetFramework>
+  </PropertyGroup>
+
+  <!-- Build information -->
+  <PropertyGroup>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <AssemblyOriginatorKeyFile>../../Gax.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>
+    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+  </PropertyGroup>
+
+  <!-- Packaging information -->
+  <PropertyGroup>
+  	<Copyright>Copyright 2017 Google Inc.</Copyright>
+    <Authors>Google Inc.</Authors>
+    <IncludeSymbols>True</IncludeSymbols>
+    <PackageLicenseUrl>https://developers.google.com/open-source/licenses/bsd</PackageLicenseUrl>
+    <PackageProjectUrl>https://github.com/googleapis/gax-dotnet</PackageProjectUrl>
+    <PackageTags>Google</PackageTags>
```

Comment:
It won't as far as I know, but everything in this project just uses the same tag. If that's ever not the case, we can break things out later. I suspect it could be merged with something like

```xml
<PackageTags>$(PackageTags);Foo</PackageTags>
```

but I haven't tried that.